src/Resources/ProductCategories.php

```diff
@@ -17,19 +19,66 @@ public function __construct(Connection $connection)
         $this->connection = $connection;
     }
 
+    // this is only returning information for the root object for now
     public function get()
     {
-        return $this->connection->get('/ProductCategories');
+        $response = $this->connection->get('/ProductCategories');
+        $body = $response->getBody();
+        return $this->getResponseFromBody($body);
     }
 
     public function post(ProductCategory $productCategory)
     {
         $payload = $productCategory->toPost();
+        $parentId = $productCategory->getParentId();
+        $urlExtension = $parentId == -1 ? '' : '/' . $parentId;
```

Comment:
Yes and I think there might be a bug here actually if there isn't a root category but that bug would be in IntegrationService where I'm not providing a -1 when there isn't a root category. I'll have to revisit, and I'll add comments. Kind of forgot I'd done this.